src/override/malloc.cc

```diff
@@ -43,10 +43,18 @@ extern "C"
     return ThreadAlloc::get_noncachable()->alloc<ZeroMem::YesZero>(sz);
   }
 
+#ifndef __ANDROID__
   SNMALLOC_EXPORT size_t SNMALLOC_NAME_MANGLE(malloc_usable_size)(void* ptr)
   {
     return Alloc::alloc_size(ptr);
   }
+#else
+  SNMALLOC_EXPORT
+  size_t SNMALLOC_NAME_MANGLE(malloc_usable_size)(const void* ptr)
+  {
+    return Alloc::alloc_size(const_cast<void*>(ptr));
```

Comment:
`alloc_size` shouldn't need to modify anything through the pointer.  I'd prefer to pass a `const void*` here and unify both code paths.

Reply:
Do you mean changing the inner paths within snmalloc?

Reply:
Yes, exactly.  I don't think `Alloc::alloc_size` should need to take a non-`const` pointer, so let's make that one `const`.